oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb

```diff
@@ -21,7 +21,7 @@
 
   include Pedant::RSpec::CookbookUtil
 
-  context "DELETE /cookbooks/<name>/<version>" do
+  context "DELETE /cookbook_artifacts/<name>/<version>" do
```

Comment:
It is better, and I'll fix it. Due to some conceptual and code copy-paste, the code and tests often play fast and loose with terminology.